arclightning_backend/src/config.rs

```diff
@@ -4,24 +4,35 @@ use std::io::{self, ErrorKind, Read};
 use std::path::PathBuf;
 use toml;
 
+#[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
+pub struct Config {
+    pub games_config: HashMap<String, Game>,
```

Comment:
let's change games-config to games

instead of having a separate server config, just put the struct elements in here

Reply:
I don't know how to organize the `.toml` file to have unassociated elements